docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md

```diff
@@ -0,0 +1,216 @@
+# Topic
+
+This tutorial explains how to use conversational search with Cohere Command model. Read this doc [Conversational search](https://opensearch.org/docs/latest/search-plugins/conversational-search/) for more details.
+
+Note: You should replace the placeholders with prefix `your_` with your own value
+
+# Steps
+
+## 0. Preparation
+
+1. Enable conversational search and memory feature
+```
+PUT _cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.memory_feature_enabled": true,
```

Comment:
Just reminder this can be removed after changing the default value to true.

Reply:
correct